src/nvim/tui/termkey/driver-ti.c

```diff
@@ -410,10 +410,15 @@ int start_driver_ti(TermKey *tk, void *info)
   // Can't call putp or tputs because they suck and don't give us fd control
   len = strlen(start_string);
   while (len) {
-    size_t written = (size_t)write(tk->fd, start_string, (unsigned)len);
-    if (written == (size_t)-1) {
+#ifndef _WIN32
+    ssize_t result = write(tk->fd, start_string, len);
+#else
+    ssize_t result = write(tk->fd, start_string, (unsigned int)len);
```

Comment:
what was wrong with the `(unsigned)len` cast? was coverity, or non-windows CI, complaining about it?

Reply:
The posix write signature calls for size_t and the windows write signature calls for unsigned int. I was trying to be explicit in this difference since windows ci fails if not cast to unsigned int.

Reply:
if (unsigned) doesn't give coverity/CI warnings I think it would be better to avoid the verbosity of the #ifdefs.

Reply:
fixed it!